pallets/dapp-staking-v3/rpc/runtime-api/src/lib.rs

```diff
@@ -42,6 +42,6 @@ sp_api::decl_runtime_apis! {
         fn blocks_per_era() -> BlockNumber;
 
         /// Get dApp tier assignment for the given dApp.
-        fn get_dapp_tier_assignment() -> BTreeMap<DAppId, TierId>;
+        fn get_dapp_tier_assignment() -> BTreeMap<DAppId, RankedTier>;
```

Comment:
Only the retuned value will change, interface is the same. I don't see any reason to bump.  DApps using this will just need to implement decoding and it will be backward compatible.

Reply:
True, but what the function returns _changes_, even though the type is exactly the same.
Such changes should be reflected in the version change as well because otherwise, how do you know when this change occurred. I understand in this scenario it's not as important since we have 4 tiers, and all historic entries have the upper 4 bits as zero.

Anyways, seems like a grey area to me.
No need for any changes.